common/src/api/external/mod.rs

```diff
@@ -3357,13 +3357,22 @@ pub struct TufRepoDescription {
 
     /// Information about the artifacts present in the repository.
     pub artifacts: Vec<TufArtifactMeta>,
+
+    // Association between an artifact and its RoT signing information.
+    // This also covers the RoT bootloader artifacts which are signed as well.
+    pub rots_by_sign: Vec<TufRotBySign>,
 }
 
 impl TufRepoDescription {
     /// Sorts the artifacts so that descriptions can be compared.
     pub fn sort_artifacts(&mut self) {
         self.artifacts.sort_by(|a, b| a.id.cmp(&b.id));
     }
+
+    /// Sorts the RoTs by sign so that descriptions can be compared.
+    pub fn sort_rots_by_sign(&mut self) {
+        self.rots_by_sign.sort_by(|a, b| a.id.cmp(&b.id));
+    }
 }
```

Comment:
If we do keep a separate field, we should probably have a single sort method here. Would there be a case where you want to sort one but not the other?